coalib/bearlib/aspects/__init__.py

```diff
@@ -54,6 +54,31 @@ def __init__(self, module):
                 subaspect = getattr(submod, submodname)
                 setattr(self, submodname, subaspect)
 
+    def get(self, aspectname, multi_find=False):
```

Comment:
Remove that `multi_find` feature. The method shouldn't return either a single aspect or a sequence of aspects based on the same kind of input. That's confusing and needs a check every time what you actually got. Let's maybe implement some `.get_all()` for that later...